code/code/sys/db.cc

```diff
@@ -3437,15 +3437,16 @@ void runResetCmdZ(zoneData &zone, resetCom &rs, resetFlag flags, bool &mobload,
 
 void runResetCmdY(zoneData &zone, resetCom &rs, resetFlag flags, bool &mobload, TMonster *&mob, bool &objload, TObj *&obj, bool &last_cmd)
 {
+  if (!mob || !mobload) {    
+    return; 
```

Comment:
should log an error, complete with zone name and any other helpful data.

Reply:
That function gets called with a null pointer passed to mob hundreds of times during boot, so it would be adding a lot of bloat to the logs, but I definitely can if you want. I'm guessing you saw it, but just in case - I tried to explain my thought process on this one in the comments on this commit itself.

Reply:
oof, no, then not.
Tho there's a rule of thumb that says - anything asked in code review deserves explaining in the code itself. Perhaps add a comment about that?

Reply:
Commit log is not what one usually reads, when trying to figure out what a given bit of code does :)